app/app/calculations/punching.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+from math import pi, sin, cos, ceil
+from .consts import *
+
+ERRORS = []
+
+
+def compute_punching(c_class, s_class, dsw, support, section, b, h, dx, dy, lx,
+                     ly, ad, lambda_u, asx, asy, design_situation, ved, beta):
+    """
+    Procedura sprawdzenia nośności na przebicie płyty betonowej wg PN-EN 1992-1
+    """
+
+    # -----------------------------------------
+    # MATERIAL
+    # Beton
+    fck = c_tab[c_class.type] * MPa
+    # Stal
+    fyk = s_tab[s_class.type] * MPa
+    alfa_sw = 0.8           # redukcja wytrzymalosci zbr. na ścinanie, TODO(LZ): do namyslu jaka wartosc
+    
+    # -----------------------------------------
+    # GEOMETRIA
+    # Element = 'plyta'    # do wyboru: plyta, stopa fundamentowa TODO
+    cx = float(b) * cm     # cx = 40 * cm
+    cy = float(h) * cm     # cy = 40 * cm
```

Comment:
Jak po # są przykłady, to niech będą zgodne z rzeczywistościa, tyle cyfr po przecinku, ile jest w założeniu.